bulbs/styles/static/styles/bulbs-styles/components/ads.scss

```diff
@@ -0,0 +1,8 @@
+.dfp-slot-inread[data-ad-load-state="loaded"] {
+  color: #7f7f7f;
```

Comment:
actually, the way I have it set up now, if the div with class dfp-slot-inread has a display set to none then the text won't display. the `data-ad-load-state` can be `loaded` even if no ad is returned from the server.